tests/support/expectations.js

```diff
@@ -740,9 +741,25 @@ module.exports = function (request, config, isServer) {
         request.put('https://context.delay.example/test')
           .end(function (err, result) {
             test.equal(result.data, 'test'); // just to see the arguments are passed as usual
-            test.equal(timePassed, 3000); // setTimeout has been called
+            test.equal(setTimeout.calls.length, 1); // setTimeout has been called
             test.done();
           });
+      },
+      'calling callback function after emitting progress events': function (test) {
+        var parts = 3, currentPart = 1;
+        var r = request.put('https://context.progress.example/' + parts)
```

Comment:
We are against not fully named variables ... Can you find a proper name to your `r` variable ?